src/connection.ts

```diff
@@ -360,7 +360,7 @@ export class Connection extends EventEmitter {
       responseFrames = responseFrames.concat(this.queuedFrames)
       this.queuedFrames = []
       const responsePacket = new Packet(requestPacket.sequence, IlpPacketType.Reject, prepare.amount, responseFrames)
-      this.debug(`rejecting packet ${requestPacket.sequence}: ${JSON.stringify(responsePacket)}`)
+      this.log.error(`rejecting packet ${requestPacket.sequence}: ${JSON.stringify(responsePacket)}`)
```

Comment:
debug or trace is probably better here since this doesn't mean there is an actual error happening.